bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -177,6 +177,7 @@ private void checkConfiguration() throws ConfigValidationException {
         Configuration testConfig = this.getConfig();
         String testRetryCount = testConfig.get(RETRY_COUNT).toString();
         String testRequestTimeout = testConfig.get(REQUEST_TIMEOUT).toString();
+        baseUri = testConfig.get(BASE_URL) != null ? testConfig.get(BASE_URL).toString() : null;
```

Comment:
Cjheck here that the value is not null and if it is then trigger a `ConfigValidationException` exception

Reply:
With the new config validation by the core framework done before initializing the thing handler, I am not sure this is even necessary. I mean, as you defined the parameter as required, I believe your thing handler will not be initialized if the config parameter is not set. Maybe you can check that. As a result, you would get always a non null value here and you have no check to add or exception to trigger.

Reply:
Thanks for the feedback. Updated.